tests/test_core.py

```diff
@@ -524,6 +525,15 @@ async def coroutine_listener(event):
     assert len(coroutine_calls) == 1
 
 
+async def test_eventbus_max_length_exceeded(hass):
+    """Test that an exception is raised when the max character length is exceeded."""
+
+    with pytest.raises(MaxLengthExceeded):
+        hass.bus.async_fire(
```

Comment:
Should we check `value`, `property_name`, and `max_length` are set correctly?

Reply:
hmm I'm not sure how to do that, is there a way to capture the exception as part of setting up an exception assertion like this?